src/tasks/siri-vm-generator/jobs/siri-vm-generator.ts

```diff
@@ -40,11 +40,17 @@ void (async () => {
 
         const totalAvlCount = avls.length;
         const matchedAvlCount = avls.filter((avl) => avl.route_id && avl.trip_id).length;
+        const tflTotalAvlCount = avls.filter((avl) => avl.operator_ref === "TFLO").length;
+        const tflMatchedAvlCount = avls.filter(
+            (avl) => avl.operator_ref === "TFLO" && avl.route_id && avl.trip_id,
+        ).length;
```

Comment:
I'd probably simplify to:
```suggestion
        const tflMatchedAvlCount = avls.filter((avl) => avl.operator_ref === "TFLO").length;
```